spec/migration_spec.rb

```diff
@@ -62,7 +62,7 @@
 
       it 'changes scenario answers' do
         _, after = migrator
-        expect(answers_after).to eq after
+        expect(after).to include answers_after
```

Comment:
Are these changes still needed?

Reply:
Yes. I either have to modify those fixtures which are targeting a specific chunk of the answers or make this change. This change feels more accurate.

Reply:
I'm still torn between an exact match and this. If we go this route, why don't we write more precise tests? Let's take the remove-tuning-fact example:
```diff
--- foreman-proxy-content-before.yaml	2021-06-08 11:42:35.140954220 +0200
+++ foreman-proxy-content-after.yaml	2021-06-08 11:42:35.140954220 +0200
@@ -12,8 +12,9 @@
 :hook_dirs: ['./katello/hooks']
 :parser_cache_path: ./_build/parser_cache/foreman-proxy-content.yaml
 :hiera_config: ./config/foreman-hiera.yaml
-:facts:
-  tuning: 'default'
+
+:verbose: true
+:verbose_log_level: 'notice'
 
 :order:
   - certs
```

We could also stop including an after yaml file and simply state `expect(after).not_to contain(:facts)`. The reason I chose to include complete files is that I did like to see the complete effects of a change on various answer files.

Now I'll also say I'm generally not a fan of a lot of noise in the git log so I can be persuaded. I'm asking myself: how many issues can we catch with including the full files and how many will we miss if we don't.

However, having these partial matches with files that appear complete feels deceiving to me and a sort of worst of both worlds.